play_off.py

```diff
@@ -0,0 +1,93 @@
+import collections
+import random
+
+team_list = ['FC Barcelona', 'Real Madrid', 'Liverpool FC', 'Manchester United', 'CSKA Moskva', 'Werder Bremen',
+             'Arsenal FC', 'Bayern Munchen',
+             'Chelsea FC', 'Spartak Moskva', 'Glasgow Rangers', 'Legia Warszawa', 'Olympique Lyonnais', 'IFK Mariehamn',
+             'Riga FC', 'PAOK Thessalonikis']
+
+Game = collections.namedtuple('Game', ['first_team', 'second_team', 'score_ft', 'score_sd', 'round_n'])
+
+one_eights = []
+one_fourth = []
+one_half = []
+final = []
+round_list = [one_eights, one_fourth, one_half, final]
+
+round_str = {
+    0: '1/8',
+    1: '1/4',
+    2: '1/2',
+    3: 'final',
+}
+
+
+def correct_team():
+    offer_enter = 'Enter team from list: '
+    team_input = input(offer_enter)
+    while team_input not in team_list:
+        print('ERROR!!! No such team')
+        team_input = input(offer_enter)
+    return team_input
+
+
+def different_random(random_score):
+    diff = random.randint(0, 20)
+    while diff == random_score:
+        diff = random.randint(0, 20)
+    return diff
+
+
+def winners(first_game, second_game):
+    if first_game.score_ft > first_game.score_sd:
+        winner_1 = first_game.first_team
```

Comment:
winner_1 and winner_2 is very similar.
Use first_winner and second winner instead